litestar/_parsers.py

```diff
@@ -66,25 +65,3 @@ def parse_cookie_string(cookie_string: str) -> dict[str, str]:
         )
     }
     return output
-
```

Comment:
No need for these anymore. `Headers` does the conversion itself while also caching itself to the `scope` if invoked with `Headers.from_scope`. In addition, it retains duplicate keys, which were dropped in these methods.